server/apps/rules/routing_rules.py

```diff
@@ -140,6 +143,10 @@ class RoutingRuleSchemeResource(Resource):
                             },
                             'hour_of_day_to': {
                                 'type': 'string'
+                            },
+                            'time_zone': {
+                                'type': 'string',
+                                'nullable': True
```

Comment:
I wanted to avoid breaking the existing data that does not have this info, and any other code that might be using this schema and not sending the time zone name (it's basically the same way the things have worked until now, the times were stored in UTC without explicitly mentioning the time zone).

What do others think, is it worth breaking the backward compatibility in this case? I can adjust the schema of course.